apiserver/Makefile

```diff
@@ -66,29 +77,37 @@ clear-local-apiserver: clean-cluster ## Clear local apiserver.
 
 .PHONY: fmt
 fmt: ## Run go fmt against code.
-	go fmt ./...
+	@echo "Running go fmt..."
+	@go fmt ./...
 
 .PHONY: vet
 vet: ## Run go vet against code.
-	go vet ./...
+	@echo "Running go vet..."
```

Comment:
curious why this change?
IIUC, `@` is used to suppress output for the command itself, but you echo it anyway...